lib/commands/log.js

```diff
@@ -24,6 +24,7 @@ extensions.startLogCapture = async function () {
       udid: this.isRealDevice() ? this.opts.udid : undefined,
       showLogs: this.opts.showIOSLog,
       realDeviceLogger: this.opts.realDeviceLogger,
+      xcodeVersion: this.xcodeVersion,
```

Comment:
how this is going to work if `this.xcodeVersion` is an empty object?

Reply:
It's only an empty object on real devices when `webDriverAgentUrl` is set. In the logger, the `xcodeVersion` is only used for simulators.

Reply:
it might be handy to document this fact somewhere in the logger class